docs/core/tools/csproj.md

```diff
@@ -19,10 +19,25 @@ see [the MSBuild project file](https://docs.microsoft.com/visualstudio/msbuild/m
 
 ## Implicit package references
 Metapackages are now implicitly referenced based on the target framework specified in the `<TargetFramework>` or `<TargetFrameworks>` property of your project file. 
-If the target framework is `netcoreap1.x`, the proper version of the `Microsoft.NETCore.App` metapackage is referenced. 
+If the target framework is `netcoreapp1.x`, the proper version of the `Microsoft.NETCore.App` metapackage is referenced. 
 Otherwise, if the target framework is `netstandard1.x`, the proper version of the `NetStandard.Library` metapackage is referenced.
 
-As far as the rest of the behavior is concerned, the tools will work as expected and most of the gestures will remain the same (for example, `dotnet restore`). 
+As far as the rest of the behavior is concerned, the tools will work as expected and most of the gestures will remain the same (for example, `dotnet restore`).
+
+```xml
+<PropertyGroup>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+</PropertyGroup>
+```
+
+```xml
+<PropertyGroup>
+    <TargetFrameworks>netcoreapp1.1;net462</TargetFrameworks>
+</PropertyGroup>
+```
+
+> [!WARNING]
+> `<TargetFrameworks>` is ignored if `<TargetFramework>` is specified, indepenently from order. 
```

Comment:
nit: spelling of "indepenently" :point_right: "independently" ... but ...
I'd say "... is specified, independent of order."

I wonder if this text should just be placed in-line at the end of this sentence:

> Metapackages are now implicitly referenced based on the target framework specified in the `<TargetFramework>` or `<TargetFrameworks>` property of your project file.

and nit: You might want to remove "now" where the original author had that ... history/time doesn't really mean much here imo.

making it ...

> Metapackages are implicitly referenced based on the target framework specified in the `<TargetFramework>` or `<TargetFrameworks>` property of your project file. `<TargetFrameworks>` is ignored if `<TargetFramework>` is specified, independent of order.

Your note here is important and flows right after the thought expressed there. What do you think?

I have one more comment about the paragraph structure in this section, but let's see how it looks after another commit.